install/kots/manifests/gitpod-log-collector.yaml

```diff
@@ -74,6 +74,13 @@ spec:
               Path /gitpod/log-collector
               Format plain
 
+          [OUTPUT]
+              Name file
+              Match migrations*
```

Comment:
What exactly are we matching here? The pod name?

Reply:
Yes - the file is in the format `migrations-<id.<container-name>`, so will pick up both `db-waiter` and `migrations` for the container name

Reply:
Awesome, thanks for the info!